### PR TITLE
September landing page for French locale (#9424)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
http://localhost:8000/fr/firefox/unfck/

## Issue / Bugzilla link
#9424

## Testing
Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/198109492

### CODE REVIEW

**Comment by alexgibson** [comment]
This needs to be reviewed ready for merge/push on Tuesday. Launch is Wednesday morning (EU time).

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -658,6 +658,8 @@ def firefox_channel(*args, **kwargs):
     redirect(r'^firefox/unfuck/?$', 'firefox.campaign.unfck.index'),
     redirect(r'^firefox/love/?$', 'firefox.campaign.unfck.index'),
     redirect(r'^firefox/liebe/?$', 'firefox.campaign.unfck.index'),
+    redirect(r'^firefox/rendonslenetplusnet/?$', 'firefox.campaign.unfck.index'),
```
Comment (stephaniehobson) [inline bedrock/firefox/redirects.py:661]: This forwards me to the en-US page, anything easy we can do about that?

File: bedrock/firefox/templates/firefox/campaign/unfck/index.fr.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+{% block page_desc %}Quand les données personnelles deviennent une monnaie d’échange, il est temps de choisir un navigateur qui vous protège et protège le Net que vous aimez.{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/campaign/unfck/fr/og.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {{ css_bundle('firefox-unfck-fr') }}
+{% endblock %}
+
+{% set tw_text = '2020 est peut être foutu mais cela n’est pas une raison pour qu’Internet le soit. La bonne nouvelle est qu’on peut tous aider à faire bouger les choses'|urlencode %}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.fr.html:17]: 279 characters 😅 well done EU team.

File: bedrock/firefox/templates/firefox/campaign/unfck/index.fr.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        alt_text='Deux mains se font un high five célébrant le net plus net.',
+        include_cta=True,
+        share_text='Partager',
+        download_text='Télécharger le GIF',
+        copy_text='Copier le lien du GIF',
+        copy_success_text='Lien copié'
+        ) %}
+        <form class="c-tell" action="{{ 'https://www.twitter.com/intent/tweet?url=%s&hashtags=%s&via=%s'|format(tw_url, tw_hashtags, tw_via)|e }}" method="post">
+          <textarea class="c-tell-input" placeholder="vous y voyez encore clair sur le Net ?" required maxlength="280"></textarea>
+          <button type="submit" class="c-tell-button mzp-c-button mzp-t-secondary js-tell-button">Partager</button>
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.fr.html:130]: Looks like this is using the DE short link and Twitter handle:

Screen Shot 2020-10-05 at 14 33 09

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/redirects.py:661]: It should respect language preference same as the rest of the redirects

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Something's up with the pre-filled Twitter share text. Everything else looks great.

**Comment by alexgibson** [comment]
@stephaniehobson thanks for the review. I moved the macro to an include, so that each locale can set things explicitly. I pushed the changed to demo for easier testing:

- https://www-demo1.allizom.org/fr/firefox/unfck/
- https://www-demo1.allizom.org/de/firefox/unfck/

**Comment by alexgibson** [comment]
Ok, I'm hearing this page may not launch until Monday 10th now. I'll update when I hear confirmation, but let's keep the do-not-merge label on for now.

**Comment by alexgibson** [comment]
Ok, the launch time is 8am CET on Monday 10th. We're ok to push the landing page live on Friday so it's ready.

**Review by stephaniehobson — APPROVED** [review]
R+ 🙆‍♀️

**Comment by alexgibson** [comment]
There's a circleCI failure here that's unrelated to the PR. I ran tests locally and they all passed. I also did one final push for a test run: https://gitlab.com/mozmeao/www-config/-/pipelines/200347957

**Pull request merged**